chapters/package-information.md

```diff
@@ -708,7 +708,7 @@ Provide an independently reproducible mechanism that permits unique identificati
 
 ### 7.10.2 Intent
 
-Eliminate confusion over which version or modification of a specific package the SPDX document references by providing a unique identifier of the package.
+Verify, based on the binary blob, which version or modification of a specific package the SPDX document references. This verification enables a recipient to determine if any bits in the original package have been changed but does not specify which bits.
```

Comment:
Propose changing `Verify, based on the binary blob, which version or modification of a specific package the SPDX document references.` to `Verify, based on the binary blob, which version or modification of a specific package content the SPDX document references.`

Same reasoning as above